website/addons/forward/model.py

```diff
@@ -18,10 +18,6 @@ class ForwardNodeSettings(AddonNodeSettingsBase):
     url = fields.StringField(validate=URLValidator())
     label = fields.StringField(validate=sanitized)
     redirect_bool = fields.BooleanField(default=True, validate=True)
```

Comment:
Redirect boolean now completely removed from the add-on. Add-on will now always automatically redirect.